account_manager/src/lib.rs

```diff
@@ -21,16 +21,17 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
 }
 
 /// Run the account manager, returning an error if the operation did not succeed.
-pub fn run<T: EthSpec>(matches: &ArgMatches<'_>, env: Environment<T>) -> Result<(), String> {
+pub fn run<T: EthSpec>(matches: &ArgMatches, env: Environment<T>) -> Result<(), String> {
     match matches.subcommand() {
-        (wallet::CMD, Some(matches)) => wallet::cli_run(matches)?,
-        (validator::CMD, Some(matches)) => validator::cli_run(matches, env)?,
-        (unknown, _) => {
+        Some((wallet::CMD, matches)) => wallet::cli_run(matches)?,
+        Some((validator::CMD, matches)) => validator::cli_run(matches, env)?,
+        Some((unknown, _)) => {
             return Err(format!(
                 "{} is not a valid {} command. See --help.",
                 unknown, CMD
             ));
         }
+        None => return Err(format!("{} does not have a subcommand. See --help", CMD)),
```

Comment:
```suggestion
        None => return Err(format!("{} requires a command. See --help", CMD)),
```
I think this would align better with the error above